routes/archive.js

```diff
@@ -6,7 +6,10 @@ mc = new mcapi.Mailchimp(process.env.MAILCHIMP_API_KEY);
 
 // Variable, that keeps all campaigns 
 var campaignsCache = [];
-var cachePath = "/home/scalatimes/cache/";
+var cachePath = process.env.CACHE_PATH;
+if (!cachePath.endsWith("/")) cachePath += '/';
+console.log("Disk cache path: " + cachePath);
+var mcFetchLimit = process.env.MC_LIMIT;
```

Comment:
I'd use `process.env.MC_LIMIT || <SOME_DEFAULT_VALUE>` just in case env var is missing.